build.gradle.kts

```diff
@@ -27,6 +27,8 @@ dependencies {
 
     implementation("com.squareup.tools.build:maven-archeologist:0.0.10")
 
+    implementation("org.virtuslab:pulumi-google-native-kotlin:0.28.0.1")
```

Comment:
Why is this needed?

Reply:
Good catch. Committed by mistake. Removed!